README.md

```diff
@@ -21,6 +24,38 @@ Barnabas provides a way to run an [Apache Kafka][kafka] cluster on
 
 <!-- /TOC -->
 
+## Deploying Kubernetes/OpenShift cluster
+
+A Kafka/Zookeeper cluster can be deployed on a Kubernete or OpenShift cluster. So, first of all, such cluster is needed.
+
+### Kubernetes
+
+In order to interact with a Kubernetes cluster, be sure to have [the kubectl tool][kubectl] installed.
+If you don't have a Kubernetes cluster up and running, the easiest way to deploy it, for development purpose, is to use the [Minikube][minikube] project which provides a one node cluster in a VM. In order to do that you can just follow the installation guide which describes all the prerequisites and how to get the Minikube binaries.
+Finally, the cluster can be deployed by running
+
+```
+minikube start
+```
+
+### OpenShift
+
+In order to interact with an OpenShift cluster, be sure to have the [Openshift client tools][origin] installed.
+If you don't have an OpenShift cluster up and running, the easiest way to deploy it, for development purpose, is to use the [Minishift][minishift] project which provides a one node cluster in a VM. In order to do that you can just follow the installation guide which describes all the prerequisites and how to get the Minishift binaries.
+Finally, the cluster can be deployed by running
+
+```
+minishift start
+```
+
+The other way is to use directly the Openshift client tools for spinning a one node containerized cluster (it will run OpenShift as a Docker container on the local machine).
```

Comment:
> Another way is to use the OpenShift client tools directly to spin-up a single-node cluster. It will run OpenShift as a Docker container on the local machine.